frontend/src/models.ts

```diff
@@ -157,43 +167,76 @@ export type Subscriber = {
   avatar_url: string;
   is_setup: boolean;
   schedule_links: string[];
+  secondary_email?: string;
+  invite?: Invite;
+  time_created?: string;
+  time_deleted?: string;
+}
+
+export type WaitingListEntry = {
+  id: number;
+  email: string;
+  email_verified: boolean;
+  invite_id?: number;
+  invite?: Invite;
+  time_created?: string;
+  time_updated?: string;
 }
 
 export type Signature = {
   url: string;
-}
+};
 
 export type Error = { error: boolean|string|null };
+export type Exception = {
+  status_code?: number;
+  detail?: { msg: string }[]|ExceptionDetail;
+  headers?: any[];
+};
+export type ExceptionDetail = {
+  id?: string;
+  message?: string;
+  status?: number;
+}
```

Comment:
Since Exception references ExceptionDetail, can you move this above Exception. Thanks!

Reply:
Of course!